### ISSUE TITLE
Blank screen when trying to connect to viewer

### ISSUE THREAD

**Issue by Kapilthakur1520** [nerfstudio-project/nerfstudio#3462]
When training a model or loading a checkpoint, the given viewer address  lead to blank page

I'm using window 11
rest is working fine

getting the same with both pip install and when installing from source
error i'm getting is this
index-BuQG4so-.js:1 Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "text/plain". Strict MIME type checking is enforced for module scripts per HTML spec.
![image](https://github.com/user-attachments/assets/b2cbaf00-8fc3-4f29-b943-a1d6ad34b3a3)

![image](https://github.com/user-attachments/assets/1582ef15-8e53-42eb-9db3-14d7ad46a01a)

**Comment by amountnothing** [comment]
Remind: u need to add --vis viewer_legacy to command to enable legacy viewer.

**Comment by Knni9ht** [comment]
> https://viewer.nerf.studio/versions/23-05-15-1/?websocket_url=ws://localhost:7007
> 
> working fine when i'm using the legacy viewer

it worked for me

**Comment by JUNSAN-17** [comment]
> https://viewer.nerf.studio/versions/23-05-15-1/?websocket_url=ws://localhost:7007
> 
> 当我使用旧版查看器时，它运行良好

Can I ask which version you're using?

**Comment by Ben-Mack** [comment]
I'm still having this error in Windows, latest nerfstudio & gsplat installed by directly pointing to github repo

**Comment by JUNSAN-17** [comment]
Changing to an older version of nerfstuido effectively solves this problem, I'm using 3.1.3 now!

**Comment by shuiguoWall** [comment]
> Changing to an older version of nerfstuido effectively solves this problem, I'm using 3.1.3 now!
Hello, I noticed that the latest version released officially is 1.1.5, and there is no version 3.1.3 in the previous releases. Did you perhaps make a typo, or am I misunderstanding something? I’m currently experiencing an issue with the visualization website being blank, and I would greatly appreciate it if you could clarify this for me. Thank you!

**Comment by JUNSAN-17** [comment]
> > Changing to an older version of nerfstuido effectively solves this problem, I'm using 3.1.3 now!
> > Hello, I noticed that the latest version released officially is 1.1.5, and there is no version 3.1.3 in the previous releases. Did you perhaps make a typo, or am I misunderstanding something? I’m currently experiencing an issue with the visualization website being blank, and I would greatly appreciate it if you could clarify this for me. Thank you!

Yes, you need to update the version of nerfstuido to 3.1.3. The corresponding viewer version is 1.1.3.
